openpype/hosts/houdini/api/lib.py

```diff
@@ -474,6 +474,11 @@ def maintained_selection():
 def reset_framerange():
     """Set frame range to current asset"""
 
+    # Set new scene fps
+    fps = get_asset_fps()
+    print("Setting scene FPS to {}".format(int(fps)))
+    set_scene_fps(fps)
```

Comment:
Since the asset document is retrieved below you might be best of allowing `get_asset_fps` to also take the asset document as argument and use it slightly down below to make sure you don't redundantly query the database twice for the asset's document.

While you're at it - it might be worth to change the `get_asset_fps` function to also filter the database query to only `data.fps` field. Something like this:

```python
def get_asset_fps(asset_doc=None):
    if asset_doc is None:
        asset_doc = get_current_project_asset(fields=["data.fps"])
    return asset_doc["data"]["fps"]
```